data/parsing-serializing-toml.ts

```diff
@@ -8,7 +8,7 @@
  *
  * TOML is a widely used configuration language designed to be feature-rich and intuitive to write.
  */
-import { parse, stringify } from "$std/encoding/toml.ts";
+import { parse, stringify } from "$std/toml/mod.ts";
 
```

Comment:
Off topic: I wish there was a way for CI to check for `@deprecated` import statements.